starter-code/styles/modules.css

```diff
@@ -58,6 +58,19 @@
   text-decoration: none;
 }
 
+
+@media screen and (min-width: 640px) {
+  .icon-menu {
+    display: none;
+  }
+  .main-nav ul {
+    display: block;
+  }
+  .main-nav li {
+    display: inline;
```

Comment:
You should add some padding to your li, per the TODO. Nice work putting all of the list items in one row!